docs/source/use_cases/discover_public_data/dream_smc_dna.rst

```diff
@@ -31,7 +31,7 @@ This dataset comprises the three public synthetic tumor/normal pairs created for
 
 Google Cloud Platform data locations
 ------------------------------------
-* Google Cloud Storage folder `gs://public-dream-data/ <https://console.developers.google.com/storage/browser/public-dream-data/>`_
+* Google Cloud Storage folder `gs://public-dream-data/ <https://console.console.google.com/storage/browser/public-dream-data/>`_
 * Google Genomics dataset `337315832689 <https://developers.google.com/apis-explorer/#p/genomics/v1/genomics.datasets.get?datasetId=337315832689>`_.
```

Comment:
This one seems to still point to `v1` - is that still okay to use?  I thought `v1beta2` was the proper version.